about.html

```diff
@@ -0,0 +1,130 @@
+    rel="stylesheet" />
+  <link rel="stylesheet" href="style.css">
+  <title>About Geolex Cyber Space</title>
+</head>
+
+<body>
+  <header>
+    <!-- top social icons -->
+    <div class="social-section">
+      <div class="top-nav">
+        <a href="#" class="facebook">facebook</a>
+        <a href="#" class="twitter">twitter</a>
+        <a href="#" class="english">English</a>
+        <a href="#" class="mypage">My page</a>
+        <a href="#" class="logout">logout</a>
+      </div>
+    </div>
+
+    <nav class="nav">
+      <!-- logo -->
+      <h3 class="logo">
+        <a class="logo" href="index.html">Geolex</a>
+      </h3>
+
+      <div class="nav-menu">
+        <ul class="menu-list">
+          <li class="list"><a href="index.html">Home</a></li>
+          <li class="list"><a href="about.html"></a>About</li>
+          <li class="list"><a href="#services"></a>Services</li>
+          <li class="list"><a href="contact.html"></a>Contact</li>
+          <li class="list"><a href="#"></a>Extras</li>
+          <li class="list"><a href="#"></a>SignUp</li>
+        </ul>
+      </div>
+      <!-- Hamburger -->
+
+    </nav>
+    <div class="hamburger">
+      <div class="bar ham1"></div>
+      <div class="bar ham2"></div>
+      <div class="bar ham3"></div>
+    </div>
+  </header>
+
+  <main class="about">
+    <section class="hero" id="home">
+      <div class="top-txt">
+        <h3 class="hello-txt">
+          Hello! World of Digital Minds <br>
+        </h3>
+        <h2 class="title">
+          Geolex Cyber Space 2022 <br>
+          Protecting Data through knowledge sharing
+        </h2>
+        <p class="p2">
+          This year's webinar is focussing on how Big data has affected Data Security and Cyber crimes.
+          How can we protect ourselves from falling victim to cyber crime as active members of the digital community
+        </p>
+      </div>
+
+      <p class="address">please contact us per Email for furthure questions about Geolex Summit 2022 <br>
+        geolex@cyberspace.org
+      </p>
+
+      <div class="theme">
+        <h2 class="webinar-theme">
+          Geolex Cyber Space <br>
+          Digital Safe Webinar 2022 Logo
+        </h2>
+      </div>
+
+      <div class="webinar-logo">
+        <h2 class="logo2"><i class="fa-brands fa-microblog"> Geolex</i> <br> Cyber-space Webinar </h2>
+        <p class="details">18th-20th Oct 2015 |Lusaka, Zambia</p>
+      </div>
+
+    </section>
+    <section class="past-webinar">
+      <h2 class="see-past"> Highlights of our previous webinars</h2>
+
+      <p class="address">
+        Here are a few photos of our 2 previous webinars in Livingstone.
+      </p>
+
+      <div class="see-pix">
+        <h2 class="prev">2021</h2>
+        <p class="see-details">cyberspace webinar in Lusaka</p>
+      </div>
+      <div class="see-pix2">
+        <h2 class="prev">2020</h2>
+        <p class="see-details">cyberspace webinar in Kitwe</p>
+      </div>
```

Comment:
- [ ] Kindly try to follow the design guidelines as much as possible, for instance, these 2 divs are supposed to sit side by side as a row and not as a column. Hint: you can use flexbox to align them 💯 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/190703297-0485e2ba-44a1-4bd8-9d0d-19611aa9f1c9.png)

## Guidelines:
![image](https://user-images.githubusercontent.com/50721479/190703470-6f9da8ec-423d-4333-89d4-345635bb8bcb.png)

Reply:
i have sorted this out. thank you